### DISCUSSION TITLE
12-11

### DISCUSSION THREAD

**Discussion by oauch** [ChristmasTest/christmas#65] [Daily_Meeting]
### 각자 개발 해온 거 발표
- 슬희님 : 크로마틱 액션, 코드 커버리지 액션
- 수영님 : 몽고DB 세팅
- 윤혁님 : Framer-motion 리팩토링
- 수님 : 아직 종강시즌 🚨
<br />

### 공유할 내용 있으면 하기
- 없음

<br />

###  역할 정하기
- 슬희님 : 리드마 작성, modal 디자인
- 수영님 : 다시하기 기능
- 윤혁님 : footer
- 수님 : 테스트 페이지, 결과 페이지 마무리

<br />

### PR 한번씩 확인하고 머지 작업하기?
🚨🚨🚨

### ⛄️ TMI

- 슬희님 : 
- 수영님 : 
- 윤혁님 : 
- 수님 :

**Comment by hanseulhee** [comment]
머지할 거 산더미다 산더미!!
